lmorpho/lmorpho.cpp

```diff
@@ -0,0 +1,113 @@
+#include <algorithm>
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <sstream>
+#include <vector>
+
+#include <tinyopt.hpp>
+#include <util/optional.hpp>
+
+#include "morphology.hpp"
+#include "morphio.hpp"
+#include "lsystem.hpp"
+#include "lsys_models.hpp"
+
+namespace to = nest::mc::to;
+using nest::mc::util::optional;
+using nest::mc::util::nothing;
+using nest::mc::util::just;
+
+const char* usage_str =
```

Comment:
Added a `README.md` with a lot more detail, but there is still room there for detailed usage information and e.g. sample output.